models/zeroex/celo/zeroex_celo_api_fills_deduped.sql

```diff
@@ -0,0 +1,102 @@
+               ELSE 1 END AS                                                           taker_consider_flag--from
+         , CASE
+               WHEN a.maker_token = b.taker_token AND a.maker_token_amount_raw = b.taker_token_amount_raw THEN 0
+               ELSE 1 END AS                                                           maker_consider_flag
+         , SUM(CASE
+                   WHEN a.maker_token = b.taker_token AND a.maker_token_amount_raw = b.taker_token_amount_raw THEN 0
+                   ELSE 1 END) OVER (PARTITION BY a.tx_hash ORDER BY a.evt_index DESC) hop_count
+         , a.*
+    FROM  fills_with_tx_fill_number a
+        LEFT JOIN fills_with_tx_fill_number b ON (a.tx_hash = b.tx_hash AND a.tx_fill_number = b.tx_fill_number - 1)
+        LEFT JOIN fills_with_tx_fill_number c ON (a.tx_hash = c.tx_hash AND a.tx_fill_number = c.tx_fill_number + 1)
+)
+, deduped_bridge_fills
+AS
+(
+    SELECT   tx_hash
+        , MAX(evt_index)                                                                    AS evt_index
+        , MAX(affiliate_address)                                                            AS affiliate_address
+        , MAX(CASE WHEN taker_consider_flag = 0 THEN NULL ELSE taker_token END )            AS taker_token
+        , MAX(CASE WHEN maker_consider_flag = 0 THEN NULL ELSE maker_token END )            AS maker_token
+        , MAX(CASE WHEN taker_consider_flag = 0 THEN NULL ELSE taker_symbol END )           AS taker_symbol
+        , MAX(CASE WHEN maker_consider_flag = 0 THEN NULL ELSE maker_symbol END )           AS maker_symbol        
+        , MAX(CASE WHEN taker_consider_flag = 0 THEN NULL ELSE taker_token_amount END)      AS taker_token_amount
+        , MAX(CASE WHEN maker_consider_flag = 0 THEN NULL ELSE maker_token_amount END)      AS maker_token_amount
+        , MAX(CASE WHEN taker_consider_flag = 0 THEN NULL ELSE taker_token_amount_raw END)  AS taker_token_amount_raw
+        , MAX(CASE WHEN maker_consider_flag = 0 THEN NULL ELSE maker_token_amount_raw END)  AS maker_token_amount_raw
+        , COUNT(*)                                                                          AS fills_within
+    FROM fills_first_last a
+    GROUP BY  tx_hash,hop_count
+)
+SELECT  a.blockchain
+      , '0x API'  as project
+      , cast(null as varchar(10)) as version
+      , a.block_date
+      , a.block_time
+      , b.taker_symbol AS taker_symbol
+      , b.maker_symbol AS maker_symbol
+      , CASE
+        WHEN LOWER(b.taker_symbol) > LOWER(b.maker_symbol)
+        THEN CONCAT(COALESCE(b.maker_symbol, ''), '-', COALESCE(b.taker_symbol, ''))
+        ELSE CONCAT(COALESCE(b.taker_symbol, ''), '-', COALESCE(b.maker_symbol, ''))
+        END AS token_pair   
+      , b.taker_token_amount
+      , b.maker_token_amount
+      , CAST(b.taker_token_amount_raw AS DECIMAL(38,0)) AS taker_token_amount_raw
+      , CAST(b.maker_token_amount_raw AS DECIMAL(38,0)) AS maker_token_amount_raw
```

Comment:
Maybe without casting to decimal(38,0) is ok.
```suggestion
      , b.taker_token_amount_raw AS taker_token_amount_raw
      , b.maker_token_amount_raw AS maker_token_amount_raw
```